src/components/BitcoinIcon.jsx

```diff
@@ -4,7 +4,10 @@ import { SvgIcon } from "@material-ui/core";
 import { MAINNET } from "unchained-bitcoin";
 
 const BitcoinIcon = ({ network }) => (
-  <SvgIcon htmlColor={network === MAINNET ? "orange" : "grey"}>
+  <SvgIcon
+    data-testid="bitcoin-icon"
```

Comment:
So in order for the testing library to be effective, will every component we want to test have to have a data-testid field added like this?

Reply:
Also, I'm not sure I have a good answer to this but is it desirable to have html attributes that are only for testing that appear in the built code? Still need to test this in a prod build, but in my local html I get this:

```
<svg class="MuiSvgIcon-root" focusable="false" viewBox="0 0 24 24" color="grey" aria-hidden="true" data-testid="bitcoin-icon">
```

Reply:
> So in order for the testing library to be effective, will every component we want to test have to have a data-testid field added like this?

No, you can also query by [other methods](https://testing-library.com/docs/dom-testing-library/api-queries#queries) (ByLabelText, ByPlaceholderText, ByText, ByAltText, ByTitle, ByDisplayValue, and ByRole).  I suppose I could maybe use alt text here potentially if `<SvgIcon>` supports it.

> Also, I'm not sure I have a good answer to this but is it desirable to have html attributes that are only for testing that appear in the built code? 

I personally don't see how it hurts anything, it's not being used for anything else other than tests and it's not like we're rendering semantic html anyway.